packages/builder/src/pages/builder/portal/manage/users/_components/BasicOnboardingModal.svelte

```diff
@@ -0,0 +1,58 @@
+<script>
+  import { ModalContent, Body, Input, notifications } from "@budibase/bbui"
+  import { createValidationStore, emailValidator } from "helpers/validation"
+  import { users } from "stores/portal"
+
+  const [email, error, touched] = createValidationStore("", emailValidator)
+  const password = generatePassword()
+
+  function generatePassword() {
+    return Array(30)
+      .fill(
+        "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz~!@-#$"
+      )
+      .map(
+        x =>
+          x[
+            Math.floor(
+              (crypto.getRandomValues(new Uint32Array(1))[0] /
+                (0xffffffff + 1)) *
```

Comment:
Nice short way of generating a (more secure) password, but the indentation hurts my soul haha. Security probably isn't too much of a concern since these are internal tools and it's only an initial password, so we could also just use something quick like `Math.random().toString(36).substr(2, 10)` for a 10 character password.

Reply:
Haha! It's my go-to password generating snippet I found a while back. Sure, I can swap it out to something less.... crazy 🥳